config/storage.yml

```diff
@@ -4,11 +4,3 @@ local: &local_storage
 
 test:
   <<: *local_storage
```

Comment:
technically this whole file can be deleted
dependency need to be removed in application.rb